src/analytics/Properties.tsx

```diff
@@ -1141,6 +1141,10 @@ interface DappExplorerEventsProperties {
   [DappExplorerEvents.dapp_unfavorite]: DappProperties
   [DappExplorerEvents.dapp_open_info]: undefined
   [DappExplorerEvents.dapp_open_more_info]: undefined
+  [DappExplorerEvents.dapp_filter]: {
+    id: string
+  }
+  [DappExplorerEvents.dapp_filter_remove]: undefined
```

Comment:
i think it'd be nice to combine these two events, then you could also track more specifically which dapp filter is selected and removed? the event property could take the dapp id + a `isApplied` boolean?